Sources/Basics/EnvironmentVariables.swift

```diff
@@ -11,35 +11,38 @@
 //===----------------------------------------------------------------------===//
 
 import class Foundation.ProcessInfo
+import typealias TSCBasic.ProcessEnvironmentBlock
+import struct TSCBasic.ProcessEnvironmentKey
+import enum TSCBasic.ProcessEnv
 
-public typealias EnvironmentVariables = [String: String]
+public typealias EnvironmentVariables = ProcessEnvironmentBlock
```

Comment:
Do we need to worry about this accidentally causing build issues?

Reply:
I don't think we expose it anywhere outside of SwiftPM to any clients, at least on purpose. IIUC corresponding TSC APIs were widely adopted instead. As soon as `package` support is merged in my other PR, I'll try using `package` visibility here to prevent any unintended adoption of this typealias in the future.

Reply:
Could use SPI in the meantime, would also make it easy to remember all the places we want `package` 😅

Reply:
And can now use `package`.